src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -55,7 +55,7 @@ class IdentityUserAvatar extends React.Component<Props, State> {
       <>
         <div className="user-popover-header">
           <div className="user-popover-header-name">
-            {firstName} {lastName}
+            {firstName ?? null} {lastName ?? null}
```

Comment:
Edit is to avoid [rendering unnecessary DOM nodes](https://stackoverflow.com/questions/30097091/correct-way-to-define-an-empty-dom-element-in-react#:~:text=If%20you%20use%20an%20empty,so%20it's%20not%20the%20same.&text=Bump%3A%20per%20react%20v.) when `firstName` and `lastName` are empty strings.